api/tests/unit/domain/usecases/get-certification-details_test.js

```diff
@@ -5,34 +5,26 @@ const CertificationAssessmentStates = require('../../../../lib/domain/models/Cer
 
 describe('Unit | UseCase | get-certification-details', function() {
 
-  const certificationAssessmentRepository = {
-    // TODO: Fix this the next time the file is edited.
-    // eslint-disable-next-line mocha/no-setup-in-describe
-    getByCertificationCourseId: sinon.stub(),
-  };
-
-  const placementProfileService = {
-    // TODO: Fix this the next time the file is edited.
-    // eslint-disable-next-line mocha/no-setup-in-describe
-    getPlacementProfile: sinon.stub(),
-  };
-
-  const competenceMarkRepository = {
-    // TODO: Fix this the next time the file is edited.
-    // eslint-disable-next-line mocha/no-setup-in-describe
-    findByCertificationCourseId: sinon.stub(),
-  };
-
-  const scoringCertificationService = {
-    // TODO: Fix this the next time the file is edited.
-    // eslint-disable-next-line mocha/no-setup-in-describe
-    calculateCertificationAssessmentScore: sinon.stub(),
-  };
```

Comment:
Pour moi cette partie pourrait rester ici. 
Il faudrait dans ce cas mettre `no-setup-in-describe` sur le bloc

Reply:
@aceol C'est ok de désactiver les règles de linter ?